src/qt/forms/optionsdialog.ui

```diff
@@ -238,7 +238,7 @@
              <string>Whether to show coin control features or not.</string>
             </property>
             <property name="text">
-             <string>Enable coin &amp;control features</string>
+             <string>Enable co&amp;in control features</string>
```

Comment:
This makes the shortcut hard to see, it might as well not be there if you can't reasonably see it. Additionally, the letter `i` seems off for representing `Enable coin control features`. Maybe consider if there is a letter here that can make the shortcut more visible and feel more appropriate for the feature?

Reply:
I think the visibility is slightly affected, so I agree with your suggestion of using another letter.

However, all the first intuitive first choices for this option, `E`, `c`, `o`, `f`, are used by other options.

I think `a` would be a good choice for this feature. Though it is not the most intuitive letter, it will help with the visibility as this would at the starting of the option name.

`En&amp;able coin control features`